NOTE: One should not participate in such activity and the author has not participated in such activity. 
This is based on that one movie about friends who cheat on the SAT. 
A group of four friends are about to take a major international math competition exam similar to the IMO but which is made up of one multiple choice question in mathematics. They pass by a room and see the Head of Examination for their school discuss the answer. Simon says he overheard the Head saying the answer is A and thinks it is the answer. The friends, since they love math, decided to do an analysis. The distribution of answers in the past are as below:
A: 35% 
B: 19% 
C: 20% 
D: 36% 
Given that Simon was telling the truth  (in that he indeed heard A), and human hearing at age 18 (Simon’s age) is inaccurate 0.24 of the time, what is the probability that the answer is indeed A? 


In [6]:
from __future__ import print_function, division

from thinkbayes2 import Hist, Pmf

% matplotlib inline

In [10]:
pmf = Pmf()
pmf.Set('The answer to the exam is indeed A', 0.35)
pmf.Set('The answer is not A', 0.65)
pmf.Mult('The answer to the exam is indeed A', 0.76)
pmf.Mult('The answer is not A', 0.24)
pmf.Normalize()
pmf.Print()

The answer is not A 0.369668246445
The answer to the exam is indeed A 0.630331753555


Explanation: The prior is stated as above. The original question is : given Simon heard A, what is the probaiblity that the answer is indeed A? To answer this question, H is the event that the answer is indeed A, and D is the event that P(H1) = 0.35, where H1 is the answer is indeed A. P(H2) = 0.65, where H2 is that the answer is not A. The likelihood, or P(D|H1) = (0.35)(0.76), and P(D|H2) = (0.65)(0.24). Thus, by adding the two likelihoods and finding the probability, the probability of the answer to the exam being indeed A is 0.63. 